src/rebar_prv_dialyzer.erl

```diff
@@ -489,13 +490,15 @@ run_dialyzer(State, Opts, Output) ->
     %% NOTE: Option `timing` accept boolean() or 'debug', but here we support
     %% only boolean().
     Timing = proplists:get_bool(statistics, Args),
+    CallgraphFile = proplists:get_value(dump_callgraph, Args, ""),
     %% dialyzer may return callgraph warnings when get_warnings is false
     case proplists:get_bool(get_warnings, Opts) of
         true ->
             WarningsList = get_config(State, warnings, []),
             Opts2 = [{warnings, legacy_warnings(WarningsList)},
                      {check_plt, false},
-                     {timing, Timing} |
+                     {timing, Timing},
+                     {callgraph_file, CallgraphFile} |
```

Comment:
what happens when the callgraph_file is empty? Where does the info get dumped?

Reply:
Empty `callgraph_file` means no `callgraph_file`, so no callgraph file in result